man/git-fork.md

```diff
@@ -16,9 +16,11 @@ git-fork(1) -- Fork a repo on github
   If a url is not given and the current dir is a github repo, fork the repo.
 
   1. forks the current repo
-  3. rename the `origin` remote repo to `upstream`
+  2. rename the `origin` remote repo to `upstream`
   3. adds the forked repo as a remote called `origin`
 
+  Remotes will use ssh if you have it configured with GitHub, if not, https will be used.
```

Comment:
Maybe change this to:

The `ssh` remote will be used if you have used an `ssh` remote with GitHub in the past. Otherwise, the `https` remote will be used.